ipywebrtc/webrtc.py

```diff
@@ -39,12 +39,31 @@ class MediaStream(DOMWidget):
 # for backwards compatibility with ipyvolume
 HasStream = MediaStream
 
+
+@register
+class WidgetStream(MediaStream):
+    """Represents a widget media source.
+    """
+    _model_name = Unicode('WidgetStreamModel').tag(sync=True)
+    _view_name = Unicode('WidgetStreamView').tag(sync=True)
+
+    widget = Instance(DOMWidget, allow_none=False).tag(sync=True, **widget_serialization)
+    max_fps = Int(60).tag(sync=True)
```

Comment:
There is also a difference between passing an argument and not passing an argument. We could have allow_none=True, and have None as default value.

Reply:
As you wish, I can do that if you want :) Do you agree with the fact that we should put a fps attribute for each MediaStream class ?

Reply:
Maybe in an other PR